portfolio.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html lang="en-GB">
+
+	<head>
+		<title>My Portfolio</title>
+		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
+	</head>	
+	
+	<body>
+		
+		<nav class="nav_bar">
+				<p>Karen Schulz</p>
+			
+			<div>
+				<a href="portfolio.html">Portfolio</a>
+				<a href="contact.html">Contact</a>
+			</div>
+			
+		</nav>
+		
+	
+		<div class="portfolio_page">
+			
+			<img class="mypicture" src="images/profile_pic.png" alt="my_profile_picture">
+		</div>
+			
+		<article class="content">
+			
+			<h1>Welcome to my Portfolio</h1>
+			
+			<div class="portfolio1">
+				<p>
+					<img class="first_work" src="images/green_back.png" alt="my_first_work">
```

Comment:
I dont think this image needs to nested in a p tag. just image would work i believe

Reply:
I erased the <p> tag and you are right, it was OK.